### DISCUSSION TITLE
disabling bottom of magic mouse

### DISCUSSION THREAD

**Discussion by BluefinDevelopers** [rxhanson/Multitouch-Community#111] [Q&A]
I don't know if this has been explained, have searched around and haven't seen anything . . . 

I am new to Multitouch and think I will like it.  One of my issues with the magic mouse is my palm brushes against the bottom of the mouse as i am using it and inadvertantly reads that touch causing unwanted mouse behaviors.  Is there a way to totally disable the bottom half or third  of the magic mouse so this can't happen?

**Comment by rxhanson** [comment]
Thanks for checking out Multitouch. Currently, the app is designed in such a way that it does not prevent touch information from reaching the rest of the OS. This is primarily to maximize stability, but also highlights a goal with the app: to work alongside macOS rather than to mutate the behavior of macOS (although there are some exceptions within the app). 

I don’t have plans to build this in, but I am always open to reconsidering if there is significant demand for a feature.
